remote/rtRemoteServer.cpp

```diff
@@ -443,6 +443,8 @@ rtRemoteServer::onClientStateChanged(std::shared_ptr<rtRemoteClient> const& clie
     }
   }
 
```

Comment:
Client seems to have states are ok also 
  enum class State
  {
    Started,
    Shutdown
  };

Shouldn't this be guarded with if (state == Shutdown) ?

Reply:
Hi @gladish ,
This code is not on the client but on the server side (rtRemoteServer) and it is called only when the client initiates the "Shutdown" event.